hastexo/admin.py

```diff
@@ -3,7 +3,10 @@
 """
 from django import forms
 from django.contrib import admin
-from student.models import AnonymousUserId
+try:
+    from common.djangoapps.student.models import AnonymousUserId
+except RuntimeError:
```

Comment:
I'm confused. Why exactly do we need `RuntimeError` here, instead of `ImportError`?

Reply:
I tested this in devstack and I got a RuntimeError instead of ImportError.